src/miner.h

```diff
@@ -12,11 +12,8 @@
 #include <stdint.h>
 
 class CBlockIndex;
+class CChainParams;
 class CScript;
-#ifdef ENABLE_WALLET
-class CReserveKey;
-class CWallet;
```

Comment:
The commit this is cherry-picked from does not remove `class CWallet`. I guess that this would cause a compilation error if it were an actual problem.